oshc/main/static/main/js/script.js

```diff
@@ -5,4 +5,15 @@ $(document).ready(function() {
       scrollTop: 0
     }, 1000);
   });
+
+  $(".dropdown").hover(
```

Comment:
I think the bootstrap default drop-down works fine. Let's just stick to that only :smile: